opta/core/kubernetes.py

```diff
@@ -335,6 +335,16 @@ def get_linked_secrets(layer_name: str) -> dict:
     )
 
 
+def list_pods(namespace: str = "default") -> None:
```

Comment:
why did you do list_pods? We decided on list_namespaces

Reply:
Ohh...I thought you had asked for Pods. Let me change this to namespaces.